### PR TITLE
BC-5736 - Redis for tldraw

### PR DESCRIPTION
**Pull request by wiaderwek**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-5736
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4542

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by bn-pass — APPROVED** [review]

File: ansible/group_vars/develop/cfg.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ FEATURE_SCHOOL_TERMS_OF_USE_ENABLED: "true"
 
 KEDA_NAMESPACE_ACTIVATOR_ENABLED: "true"
 
-TLDRAW_CLIENT_REPLICAS: 1
+TLDRAW_CLIENT_REPLICAS: 3
```
Comment (Loki-Afro) [inline ansible/group_vars/develop/cfg.yml:26]: this is the nginx one, right? then you don't _need_ 3 replicas on dev

Comment (davwas) [reply ansible/group_vars/develop/cfg.yml]: you are right, fixed

File: ansible/roles/dof_redis/templates/configmap.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-configmap
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: redis
+data:
+  # general
+  redis-config: client-output-buffer-limit pubsub 0 0 0
```
[the remark this answers is not in the document]
Comment (davwas) [reply ansible/roles/dof_redis/templates/configmap.yml.j2:10]: Tldraw data is being sent by redis pubsub channels. There is quite a lot of data going through and if the limit is reached redis will drop all pubsub connections causing the users between different pods working on the same board to be desynchronized.

[the remark this answers is not in the document]
Comment (mamutmk5) [reply ansible/roles/dof_redis/templates/configmap.yml.j2:10]: Okay for now it's works. Later we will change to a redis cluster.

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**